src/components/item/index.js

```diff
@@ -1,44 +1,65 @@
-import React, { Component } from 'react'
-import PropTypes from 'prop-types'
-import { Grid, Typography } from '@material-ui/core'
-import MoreMenu from 'components/moremenu'
-
-export class Item extends Component {
-  handleItemClick = option => {
-    if (option === 'Delete') {
-      this.props.deleteItem(this.props.item._id)
-    }
-  }
-
-  render () {
-    const options = [
-      'Delete'
-    ]
-    
-    return (
-      <div className="item">
-        <Grid container spacing={24}>
-          <Grid item xs={8}>
-            <Typography variant="headline" align="center">
-              {this.props.item.name}
-            </Typography>
-          </Grid>
-          <Grid item xs={4}>
-            <MoreMenu options = {options} handleItemClick = {this.handleItemClick} />
-          </Grid>
-        </Grid>
-      </div>
-    )
-  }
-}
-
-Item.propTypes = {
-  item: PropTypes.shape({
-    _id: PropTypes.string,
-    name: PropTypes.string,
-    size: PropTypes.number
-  }),
-  deleteItem: PropTypes.func
-}
-
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Grid, Typography } from '@material-ui/core'
+import MoreMenu from 'components/moremenu'
+
+import { Draggable } from 'react-beautiful-dnd'
+
+import { getItemStyle } from 'utils'
+
+export class Item extends Component {
+  handleItemClick = option => {
+    if (option === 'Delete') {
+      this.props.deleteItem(this.props.item._id)
+    }
+  }
+
+  render () {
+    const options = [
+      'Delete'
+    ]
+    
+    return (
+      <Draggable
+        key={this.props.item._id}
+        draggableId={this.props.item._id}
+        index={this.props.index}
+      >
```

Comment:
please place this on end of previous line, and all other places.

Reply:
sure.